src/bramble/client/main.js

```diff
@@ -110,13 +111,30 @@ define([
 
     // Expose Filer for Path, Buffer, providers, etc.
     Bramble.Filer = Filer;
-    var _fs = new Filer.FileSystem();
+    // We wrap the filesystem, and add metrics to keep track of project stats
+    var _fs = ProjectStats.getFileSystem();
     Bramble.getFileSystem = function() {
         return _fs;
     };
+    
     // NOTE: THIS WILL DESTROY DATA! For error cases only, or to wipe the disk.
     Bramble.formatFileSystem = function(callback) {
-        _fs = new Filer.FileSystem({flags: ["FORMAT"]}, callback);
+        _fs = ProjectStats.formatFileSystem({flags: ["FORMAT"]}, callback);
+    };
+
+    // Return total project size
+    Bramble.getTotalProjectSize = function(){
```

Comment:
These functions shouldn't go on `Bramble` but on `bramble` (i.e., on the instance).  The reason is that they are closely bound to the project root, and `bramble` is the instance created when you call `Bramble.mount()` on a project root dir.  Put it on `_instance` or in the constructor for the `BrambleProxy` (e.g., on `self`).